### DISCUSSION TITLE
Welcome to The-coder Discussions!

### DISCUSSION THREAD

**Discussion by Tangbag123** [Tangbag123/The-coder#3] [Announcements]
Hi this is Brian McBride with The-coder maintenance crew w swill now have an update log posted here so this is where you find all information on the recent updates

**Comment by Tangbag123** [comment]
I am proud to announce that the coder will have some of your favorite book in a html link in the next week

**Reply by cjscode** [reply]
I’m confused

**Reply by Tangbag123** [reply]
Why?

**Reply by cjscode** [reply]
What book?
